types/types.go

```diff
@@ -80,6 +80,8 @@ const (
 
 	LonghornLabelKeyPrefix = "longhorn.io"
 
+	LonghornVolumeAccessModeLabel = "LonghornVolumeAccessMode"
```

Comment:
```suggestion
	LonghornLabelVolumeAccessMode= "volume-access-mode"
```

Reply:
Thanks, I will follow the suggestion.
 
https://github.com/longhorn/longhorn-manager/blob/0d4593a5d1c666a1d1c8f6afeadffd2bde4a3c9f/types/types.go#L83